docs/reference/rocmcc/rocmcc.md

```diff
@@ -3,21 +3,892 @@
 ROCmCC is a Clang/LLVM-based compiler. It is optimized for high-performance computing on AMD GPUs and CPUs and supports various heterogenous programming models such as HIP, OpenMP, and OpenCL.
```

Comment:
The title should be Compiler Reference Guide.
A section with ```##``` should start with Introduction

Reply:
Fixed, moved all titles one step down the hierarchy